src/modules/utils/motordrivercontrol/MotorDriverControl.cpp

```diff
@@ -113,8 +114,9 @@ bool MotorDriverControl::config_module(uint16_t cs)
         tmc26x= new TMC26X(std::bind( &MotorDriverControl::sendSPI, this, _1, _2, _3), axis);
 
     }else{
-        THEKERNEL->streams->printf("MotorDriverControl %c ERROR: Unknown chip type: %s\n", axis, str.c_str());
-        return false;
+        chip=SPIDRVR;
```

Comment:
Sorry I missed this, the default cannot be this SPIDRVR, it MUST be explicitly set. This saves surprises for typos in config etc.

Reply:
No problem...

The raw SPI features that I added are intended to work, if invoked, for any "chip", since the feature is generic and allows for some expert debugging and tuning of any chip(s) on the SPI.  (Note that I do not actually check `chip==SPIDRVR` anywhere else.)  So, I intended for SPIDRVR to be the default.   Some of philosophical questions/options....

1. Since the raw SPI features do not depend on `chip=="SPIDRVR"`, I could get rid of the SPIDRVR enum altogether, but on second thought this is not really workable, because I want to be able to use the SPIDRVR features when I am using an otherwise undefined chip.  So, scratch this option.

1. If you prefer an error message:   If chip is not set to something recognized, then I could still have SPIDRVR be the default, but if not specifically set to "SPIDRVR", then print an error message that it is "Unknown chip type.  Defaulting to generic SPIDRVR."

1. Otherwise, if you prefer to just print an error and fail out of the module, I can just add the `switch "SPIDRVR"`, and put back the old error and fail-out.

I am partial to #2.  What do you think?

Reply:
I'd prefer the original behavior, so 3

Reply:
Done.